capabilityreceiver/testdata/cerner_capability_dstu2.json

```diff
@@ -23,7 +23,7 @@
             "security": {
                 "extension": [
                     {
-                        "url": "http://fhir-registry.smarthealthit.org/StructureDefinition/oauth-uris",
+                        "url": "http://DSTU2/fhir-registry.smarthealthit.org/StructureDefinition/oauth-uris",
```

Comment:
where did this URL come from?

Reply:
I thought this was the correct URL for the oath-uris extension for DSTU2, but I am not too sure. I realized that the other URL is the oath-uris for the R4 capability statement oath-uris extension, but since this test conformance statement is a DTSU2 resource, I figured it made more sense for it to have the correct extension for the version. I found the URL here: http://build.fhir.org/ig/Healthedata1/smart-app-launch-brver/StructureDefinition-dstu2-oauth-uris.html

Reply:
Ah okay i think you're right! I couldn't find that extension URL anywhere which is why I was confused, but yeah based on that link that seems to be correct!